src/public/V8JsiRuntime.h

```diff
@@ -78,17 +78,36 @@ struct V8RuntimeArgs {
 
   bool backgroundMode{false};
 
-  bool enableTracing{false};
-  bool enableJitTracing{false};
-  bool enableMessageTracing{false};
-  bool enableLog{false};
-  bool enableGCTracing{false};
-
-  bool enableInspector{false};
-  bool waitForDebugger{false};
-
-  // chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=localhost:9229
-  uint16_t inspectorPort{9229};
+  // Enabling all the diagnostic and tracings by default so that we will be able to tune them and writing tools over them.
+  bool trackGCObjectStats{true};
+  bool enableTracing{true};
+  bool enableJitTracing{true};
+  bool enableMessageTracing{true};
+  bool enableLog{true};
+  bool enableGCTracing{true};
+
+  // Enabling inspector by default. This will help in stabilizing inspector, and easily debug JS code when needed.
+  // There shouldn't be any perf impacts until a debugger client is attached, except the overload of having a WebSocket port open, which should be very small.

```

Comment:
>which should be very small [](start = 131, length = 26)

I think we shouldn't enable this by default for ship builds. It feels weird to open a port on customer's machines and could cause issues and questions.